lib/views/pages/home/posts/post_card/votes_widget.dart

```diff
@@ -0,0 +1,71 @@
+import "package:flutter/material.dart";
+
+/// This widget is used to display the votes of a post.
+/// It contains two buttons to upvote and downvote the post and the number of votes.
+class VotesWidget extends StatelessWidget {
+  final int votes;
+
+  const VotesWidget({
+    super.key,
+    required this.votes,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    final upvote = IconButton(
+      padding: EdgeInsets.zero,
```

Comment:
We can remove this line in my opinion

Reply:
Hello, it is used to reduce the size of the InkWell and padding created by IconButton widget. See: https://stackoverflow.com/questions/50381157/how-do-i-remove-flutter-iconbutton-big-padding

I added a note in the code thanks.